src/components/Stake/SideBar.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU, textStyle, useTheme } from '@aragon/ui'
+import CardAccordion from './CardAccordion'
+import coin from './assets/coin.svg'
+import wallet from './assets/wallet.svg'
+
+const Sidebar = React.memo(function Sidebar() {
+  return (
+    <div
+      css={`
+        display: grid;
+        grid-template-columns: 1fr;
+        grid-template-rows: auto auto auto;
+        grid-gap: ${2 * GU}px;
+      `}
+    >
+      <CardAccordion
+        card={
+          <CardContent
+            amount="72,243.47"
+            icon={wallet}
+            title="Available ANT"
+            tokenAmount="11,278.04"
+          />
+        }
+        expansion="I'm disputing this proposal because I believe that it was not made in good faith and will not benefit all ANT holders in equal measure. I'm disputing this proposal because I believe that it was not made in good faith and will not benefit all ANT holders in equal measure. ANT holders in equal measure."
```

Comment:
Do we know what the content of these cards is going to be yet? perhaps worth checking in with @dizzypaty as we'll need to have these ready to go soon.

Reply:
Do you mean the expansion? Yes, she is on it :)